_data/master/navbars/getting-started.yml

```diff
@@ -10,17 +10,7 @@ toc:
   - title: Quickstart
     path: /getting-started/kubernetes/
   - title: Requirements
-    section:
-    - title: System 
-      path: /getting-started/kubernetes/reqs/system
-    - title: Public cloud
-      section:
-      - title: Amazon Web Services
-        path: /getting-started/kubernetes/reqs/pubcloud/aws
-      - title: Azure
-        path: /getting-started/kubernetes/reqs/pubcloud/azure
-      - title: Google Compute Engine
-        path: /getting-started/kubernetes/reqs/pubcloud/gce
+    path: /getting-started/kubernetes/reqs/system
```

Comment:
The file is named "system.md" so this link is correct.

Reply:
We control the name of the file though. I was suggesting we change it.

Doesn't make sense to have two layers of requirements given it's just one page.

Reply:
We don't really have any requirements around the file names. I guess it is a small matter but it can be easier, IMO, if they have names besides index.md. I don't know, it's not a big deal, I guess but with so many index.mds it can get confusing. However, if you like it better as index.md I can rename it 😄

Reply:
YEah, I can change it I guess to match the sidebar nav